app/controllers/users_controller.rb

```diff
@@ -29,21 +29,16 @@ def create
 
   def edit
     @genres = Genre.all
-    @musician_skill = MusicianSkill.all
+    @musician_skills = MusicianSkill.all - current_user.approved_musician_skills
```

Comment:
Isn't is `current_user.pending_musician_skills + current_user.disabled_musician_skills`?
See no reasons to load all skills

Reply:
It's don't work because musician skill may be just never selected

Reply:
@Maxxx911 ok, agree.
I would recommend to create scope for the rest of skills.
```MusicianSkill.where.not(id: current_user.approved_musician_skills.pluck(:id))```